<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Vaibhav-HP" failures="1" tests="6" name="Test" time="37.115" errors="0" timestamp="2024-08-11T23:10:19 IST">
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="1.818"/>
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="1.634"/>
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="1.616">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_LoginDDT.verify_loginDDT(TC_LoginDDT.java:75)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="11.132"/>
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="3.051"/>
  <testcase classname="testCases.TC_LoginDDT" name="verify_loginDDT" time="11.217"/>
</testsuite> <!-- Test -->
